src/binaryen-c.cpp

```diff
@@ -2112,6 +2112,16 @@ BinaryenExpressionRef BinaryenSwitchGetValue(BinaryenExpressionRef expr) {
   return static_cast<Switch*>(expression)->value;
 }
 // Call
+uint8_t BinaryenCallIsReturn(BinaryenExpressionRef expr) {
```

Comment:
Not sure how important this is, but other API methods typically use `int` to denote bools.